appomni_appomni/assets/logs/appomni-appomni.yaml

```
@@ -0,0 +1,340 @@
+id: appomni-appomni
```

```suggestion
id: appomni
```
To match the manifest appID

---

Are we planning to change the id? I am not clear here

---

You can ignore this for now